docs/Build/Linux.md

```diff
@@ -16,7 +16,7 @@ Optional:
 
 ### Fedora
 ```sh
-dnf install qt5-devel
+dnf install qt5-devel qt5-qtnetworkauth-devel qt5-qtquickcontrols2-devel qt5-qtsvg-devel
```

Comment:
Funny 'qt5-devel' doesn't exist. Probably my mistake. Could you change that to 'qt5-qtbase-devel'?

Reply:
For sure. Looks like it's live there right now, but we can fix that in this PR.